test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_experiments.py

```diff
@@ -36,6 +36,7 @@
 @pytest.mark.model("mnist")
 @pytest.mark.integration("smexperiments")
 @pytest.mark.skip_py2_containers
+@pytest.mark.skip(reason="Skip this test for all images")
```

Comment:
There is already a skip marker for this test in line 35. You can remove this addition.